docs/Screenshots.stories.mdx

```diff
@@ -0,0 +1,55 @@
+<Meta
+  title="Guidelines/Screenshot testing"
+  parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+## Screenshot testing
+
+For the starting work with the screenshot test locally you need to start `storybook`:
+
+```bash
+$ npm run start
+```
+
+For the matching testing screenshots correctly, they should create in a similar OS (Ubuntu),
+to generate and update testing screenshots locally need to install the [docker](https://www.docker.com/) on the local machine and run the next command.
+
```

Comment:
suggestion:
`So that screenshot tests are matched correctly, they should be created in a similar OS (Ubuntu). In order to generate and update screenshot tests locally, you need to install [docker](https://www.docker.com/) on your local machine, then run the next command.`